lib/jormungandr/test/unit/Cardano/Wallet/Binary/JormungandrSpec.hs

```diff
@@ -72,7 +73,15 @@ genesisBlock = Block genesisHeader
         , AllowAccountCreation True
         , ConfigLinearFee (LinearFee 0 0 0)
         ]
-    , UnimplementedMessage 2
+    , Transaction $ Tx
+        { inputs = []
+        , outputs =
+            [ TxOut
+                { address = Address "3$\195xi\193\"h\154\&5\145}\245:O\"\148\163\165/h^\ENQ\245\248\229;\135\231\234E/"
+                , coin = Coin 14
```

Comment:
`Coin 14` sounds like something I've could have put in the genesis block indeed :joy: